### ISSUE TITLE
Incorrect coloring after complex var block

### ISSUE THREAD

**Issue by mokiat** [worlpaker/go-syntax#14]
VSCode colors the following incorrectly:

```go
var (
	vertexData = gblob.LittleEndianBlock(make([]byte, len(geometry.vertices)*int(stride)))
)

if 5 > 10 {

}
```

The `if` keyword is light blue instead of the standard purple.

This is a redirect of an issue opened originally to `gopls`. Please check issue for more detailed information if needed:
https://github.com/golang/vscode-go/issues/3370

**Comment by worlpaker** [comment]
Thanks for the details! This bug should be fixed in v0.6.9.

**v0.6.9**
![example](https://github.com/worlpaker/go-syntax/assets/34960716/69bdc700-48c0-4fda-8734-dfe9a64fea8f)

Can you please install the latest version of [go-syntax](https://marketplace.visualstudio.com/items?itemName=furkanozalp.go-syntax) and try it?

**Comment by mokiat** [comment]
Yes, with the latest version, after a manual install, it is no longer reproduced.

However, I am confused. I don't have this plugin installed by default. Does this mean that it ships with VSCode or with the Go plugin? Do I need to wait for those plugins to pull the latest version of this one for it to work by default (without manual install)?

**Comment by worlpaker** [comment]
Go Syntax became the default [Go grammar](https://github.com/microsoft/vscode/blob/main/extensions/go/syntaxes/go.tmLanguage.json) in the official VS Code in v1.86 ([release note](https://code.visualstudio.com/updates/v1_86#_new-go-grammar)).
 
I've published a new version (v0.6.9) to fix this issue today. VS Code updates the grammars once a month, and most likely this fix will be available in the stable VS Code next month.

Think about this extension as a pre-release, nightly version of the default VS Code Go grammar. You can either use this extension to have the quick fix, or you can wait until VS Code updates the grammars.

**Comment by mokiat** [comment]
Thanks for the quick update. It makes sense now.

I am closing this issue. Thanks, again!
